pom.xml

```diff
@@ -183,7 +183,7 @@ THE SOFTWARE.
         <artifactId>maven-surefire-plugin</artifactId>
         <!-- version specified in grandparent pom -->
         <configuration>
-          <argLine>-Dfile.encoding=UTF-8 -Xmx256m</argLine>
+          <argLine>-Dfile.encoding=UTF-8 -Xmx256m -Djava.awt.headless=true</argLine>
```

Comment:
Only relevant when running the test suite of the JTH itself. Plugins get this property from the [plugin pom](https://github.com/jenkinsci/plugin-pom/blob/fe93cf79e83f1a03e4f149b554ceaadd6f8072be/pom.xml#L50).